e2e/tests/desktop/__snapshots__/guest/cart-a11y-violations.json

```diff
@@ -0,0 +1,34 @@
+[
```

Comment:
@alexvuong Did you manually type this out or is there a way to generate these json files ?

Can we automate this in anyway and how do we do this for new pages/components added in the future ?

Reply:
yea to generate snapshots, we do `npx run test --update-snapshots`. I think it is a good idea to put that command in the npm script. Let me add that in